packages/cascader-panel/src/node.vue

```diff
@@ -148,17 +147,17 @@ export default defineComponent({
       if (isHoverMenu.value && !isLeaf.value) return
 
       if (isLeaf.value && !isDisabled.value && !checkStrictly.value && !multiple.value) {
-        handleCheck(true)
```

Comment:
Clicking the option should always check it in single-select mode, remove this line will cause another interaction bug.

Reply:
It is done, but I found another issue, It will emit close even it already checked.
Should I fix it?